tests/roles/dataplane_adoption/tasks/neutron_verify.yaml

```diff
@@ -53,6 +53,73 @@
   ansible.builtin.shell: |
       ping -c4 {{ lookup('env', 'FIP') | default('192.168.122.20', True) }}
 
+- name: tobiko validate pings, create workloads
+  when: tobiko_qe_test | default('false') | bool
+  environment:
+    OS_CLOUD_IP: "{{ standalone_ip | default(edpm_node_ip) }}"
+  block:
+    # Temporal - tobiko installation task
+    - name: tobiko installation
+      ansible.builtin.shell: |
+          ssh ${OS_CLOUD_IP} "set -o pipefail && mkdir -p ~/src/x && cd ~/src/x && git clone https://github.com/redhat-openstack/tobiko.git"
+
+    - name: oc undercloud installation
+      ansible.builtin.shell: >
+          ssh ${OS_CLOUD_IP} "curl -s -L https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz
+          | sudo tar -zxvf - -C /usr/local/bin/"
+
+    - name: copy kube conf to undercloud
+      ansible.builtin.shell: |
+        scp -r ~/.kube/ ${OS_CLOUD_IP}:~
+
+    - name: upload tobiko-playbook.yaml to the undercloud
+      delegate_to: "{{ standalone_ip | default(edpm_node_ip) }}"
+      vars:
+        tobiko_playbook: |
+          - name: Playbook to run tobiko from the undercloud pre-adoption
+            hosts: localhost
+            tasks:
+              - name: "run tests"
+                ansible.builtin.include_role:
+                  name: tobiko-run
+      ansible.builtin.copy:
+        mode: a+r
+        content: "{{ tobiko_playbook }}"
+        dest: ~/src/x/tobiko/tobiko-playbook.yaml
+
+    - name: Add tobiko.conf to the undercloud
+      delegate_to: "{{ standalone_ip | default(edpm_node_ip) }}"
+      vars:
+        tobiko_conf_file: |
```

Comment:
If we put it into the role's default variables then it could be overriden. But keeping it as-is is fine if we don't have an immediate need to customize this.